### DISCUSSION TITLE
[BUG] Unable to open commit links containing a partial commit hash

### DISCUSSION THREAD

**Discussion by amadejpapez** [community/community#15668] [Mobile] [answered]
I was reading curl's new release blog post, it had a few commit links, when clicking them, I got the app message "Something went wrong. Try again" although they opened just fine in Safari.

It appears like the app doesn't work as it should if you try to open a commit link that does not have the full commit hash.

Full commit hash, works:
`https://github.com/curl/curl/commit/620ea21410030a9977396b4661806bc187231b79`
https://github.com/curl/curl/commit/620ea21410030a9977396b4661806bc187231b79

Partial commit hash, "Something went wrong. Try again":
`https://github.com/curl/curl/commit/620ea21410030a997`
https://github.com/curl/curl/commit/620ea21410030a997

iOS 14.4.1, GitHub app 1.62.1

**Comment by stevepopovich** [comment, marked as answer]
Hey there! Interesting find. I log this bug and we will prioritize!

**Reply by amadejpapez** [reply]
Thank you! Looks like this was fixed today in 1.65.0

**Reply by stevepopovich** [reply]
Yep! Thanks for reporting.

**Comment by ghost** [comment]
![Uploading 12ED1842-2D6C-4117-8BAD-19A7ED68ABCF.jpeg…]()
